src/components/QuickSearchToolbar.tsx

```diff
@@ -6,7 +6,7 @@ import React from "react";
 
 interface QuickSearchToolbarProps {
   value: string;
-  onChange(): void;
+  onChange: React.ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
Question: Is this a related change? If not, should be removed.

Reply:
It's related to me. The max count bug was caused by switching to @material-ui/core/TextField from material-ui/TextField, which changed the typing of onChange. 

I was looking through other instances of onChange used with <TextField> and saw that this one had bad typing. Feels related enough to the core issue involved.